External/FEXCore/include/FEXCore/Utils/MathUtils.h

```diff
@@ -10,4 +14,14 @@ namespace FEXCore {
 [[nodiscard]] constexpr uint64_t AlignDown(uint64_t value, uint64_t size) {
   return value - value % size;
 }
+
+// Returns the ilog2 of a power-of-2 integer.
+// Asserts in the case that the passed in integer is not a power-of-2.
+template<typename T>
+requires(std::is_integral_v<T>)
```

Comment:
perhaps this  should be clamped to only unsigned values with `std::is_unsigned_v`

Reply:
Indeed, switched over to that.